embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/AnrSampleTest.kt

```diff
@@ -33,9 +33,8 @@ internal class AnrSampleTest {
         assertEquals(listOf(threadInfo), obj.threads)
     }
 
-    @Test
+    @Test(expected = JsonDataException::class)
```

Comment:
Do we catch/handle these new runtime exceptions so we don't introduce app crashes if something bad happens?

Reply:
I checked that all `fromJson` function invocations are surrounded by a try-catch or equivalent.